run.js

```diff
@@ -80,10 +80,14 @@ export async function run( {
 		quiet: true, // Quiet server logs during test runs
 		testUrls,
 		report: async( message ) => {
+			const reportId = message.id;
+			const report = reports[ reportId ];
```

Comment:
This change was leftover after some debugging, but it's a bit more dry so I liked it.